client/js/jsonedit.js

```diff
@@ -1788,6 +1807,12 @@ function jeLoggingRoutineEnd(variables, collections) {
   --jeLoggingDepth;
   if(!jeLoggingDepth) {
     $('#jeLog').innerHTML = jeLoggingHTML + '</div></div>';
+    // Enable the "Show Routine Debug Output" button the first time a routine is executed.
```

Comment:
Not a big deal but the button is longer called that.

Reply:
Fixed, thanks.